cudf/tests/test_csvreader.py

```diff
@@ -144,3 +144,26 @@ def test_csv_reader_skiprows_skipfooter(tmpdir):
     assert len(out.columns) == len(df_out.columns)
     assert len(out) == len(df_out)
     pd.util.testing.assert_frame_equal(df_out, out.to_pandas())
+
+
+def test_csv_reader_negative_vals(tmpdir):
+    fname = 'tmp_csvreader_file8.csv'
+
+    names = ['0', '1', '2']
+    dtypes = ['float', 'float', 'float']
```

Comment:
Should these be `float32` or `float64`?

Reply:
Specified as float32 in latest commit. Should we have a separate test for float64?